src/main/java/org/myrobotlab/service/Hd44780.java

```diff
@@ -468,13 +497,14 @@ public void clearDisplay() {
    * @param address
    */
   public void setDdramAddress(int address) {
-    if (address < 80) { // Make sure the address is in a valid range
-      lcdWriteCmd((byte) (address | 0b10000000));
-    } else {
-      error("%d Outside allowed DDRAM Address range 0 - 79", address);
+    if (address < 0 || (address > 40 && address < 63) || address > 108) { 
+      error("%d Outside allowed DDRAM Address range 0 - 108", address);

```

Comment:
Message should be updated to say the two allowed ranges, since putting in 41 will log this error even though the message says it should be fine

Reply:
updated